classrooms/views.py

```
@@ -74,3 +74,23 @@ def classroom_details(request: HttpRequest, slug: str) -> HttpResponse:
     classroom = get_object_or_404(Classroom, slug=slug)
     return render(request, "classrooms/classroom.html", {"classroom_detail": classroom})
 
+
+@login_required
+@require_http_methods(["GET"])
+def user_classroom_list(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `user_classroom_list`

(ANN201)